tests/test_number.py

```
@@ -1,7 +1,8 @@
 """Test OpenEVSE number platform."""
 
 import logging
-from unittest.mock import patch
+from asyncio import TimeoutError
```

Ruff (0.15.10)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)